include/wx/private/webviewutils.h

```diff
@@ -0,0 +1,45 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/private/webviewutils.h
+// Purpose:     Common code for wxWebView
+// Author:      Jose Lorenzo
+// Created:     2017-08-12
+// Copyright:   (c) 2017 Jose Lorenzo <josee.loren@gmail.com>
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_PRIVATE_WEBVIEWUTILS_H_
+#define _WX_PRIVATE_WEBVIEWUTILS_H_
+
+#include "wx/regex.h"
+
+// ----------------------------------------------------------------------------
+// Common wxWebView code
+// ----------------------------------------------------------------------------
+
+class wxWebViewUtils
+{
+public:
+
+    // This method is used to add a double quote level into a JavasSript code
+    // in order to get it working when eval is called programmatically.
+    static wxString WrapJavaScript
+        (const wxString& javascript, int* runScriptCount, wxString* counter)
+    {
+        *counter = wxString::Format("%i", (*runScriptCount)++);
+        wxString javascriptCopy = javascript;
+
+        // Adds one escape level if there is a single quote, double quotes or
+        // esacape characters
+        wxRegEx escapeDoubleQuotes("(\\\\*)(['\"\n\r\v\t\b\f])");
+        escapeDoubleQuotes.Replace(&javascriptCopy,"\\1\\1\\\\\\2");
+        
+        return "try { var __wx$" + *counter + " = eval(\"" +
```

Comment:
BTW, it is not necessary to reply "Done" to all the comments, this creates too many notifications to follow them all and more important ones, that require an answer, risk getting lost. Please just fix all of these trivial fixes and update the PR and leave comments more sparingly.